devimint/Cargo.toml

```diff
@@ -52,4 +52,4 @@ fedimint-portalloc = { path = "../utils/portalloc" }
 fs-lock = "0.1.3"
 
 [build-dependencies]
-fedimint-build = { version = "=0.4.0-alpha", path = "../fedimint-build" }
+fedimint-build = { path = "../fedimint-build" }
```

Comment:
Won't that break publishing? I had to painstakingly add all these version numbers in addition to paths when first publishing to crates.io :sob:

Reply:
Oh. I thought I've seen plenty of ones that didn't have it ...

Reply:
Maybe some tooling will add it automagically or I was just holding it wrong?